applications/serial_lte_modem/src/slm_ppp.c

```diff
@@ -213,7 +213,12 @@ static int ppp_start_internal(void)
 	ret = pdn_dynamic_params_get(PDP_CID, &ctx->ipcp.my_options.dns1_address,
 				     &ctx->ipcp.my_options.dns2_address, &mtu);
 	if (ret) {
-		return ret;
+		/* If any error happened on pdn getting with IPv4, try to parse with IPv6 */
+		ret = pdn_dynamic_params_get_v6(PDP_CID, &ctx->ipcp.my_options.dns1_address,
+				     &ctx->ipcp.my_options.dns2_address, &mtu);
```

Comment:
If we're not interested in these addresses on the ppp_start_internal function of SLM, shall I pass NULL for these parameters since pdn_dynamic_params_get_v6 already handling nullness.

Reply:
Yes, true, passing `NULL` should be fine then.